<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiClass</class>
 <widget class="QMainWindow" name="guiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rozpocznij</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_cpplib">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C++</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_asmlib">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Asm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyjdź</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Wątki</string>
    </property>
    <widget class="QSpinBox" name="spinBox_thread">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>81</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Biblioteka</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Plik wejściowy</string>
    </property>
    <widget class="QPushButton" name="pushButton_filein">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Wybierz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_infile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Plik wyjściowy</string>
    </property>
    <widget class="QPushButton" name="pushButton_fileout">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Wybierz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_outfile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>130</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Czas</string>
    </property>
    <widget class="QLabel" name="label_time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_savehist">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz histogram</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>pushButton_start</zorder>
   <zorder>radioButton_cpplib</zorder>
   <zorder>radioButton_asmlib</zorder>
   <zorder>pushButton_exit</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>checkBox_savehist</zorder>
   <zorder>label_status</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
</ui>
